aquia-inc/template-vite-react
# build(deps): upgrade Storybook and related dependencies to 8.2.8

## Summary

Upgraded Storybook and related dependencies to version 8.2.8.

### Added

- None

### Changed

- Upgraded `@storybook/addon-essentials` from `^7.6.20` to `^8.2.8`
- Upgraded `@storybook/addon-interactions` from `^7.6.20` to `^8.2.8`
- Upgraded `@storybook/addon-links` from `^7.6.20` to `^8.2.8`
- Upgraded `@storybook/blocks` from `^7.6.20` to `^8.2.8`
- Upgraded `@storybook/react` from `^7.6.20` to `^8.2.8`
- Upgraded `@storybook/react-vite` from `^7.6.20` to `^8.2.8`
- Upgraded `storybook` from `^7.6.16` to `^8.2.8`

### Deprecated

- None

### Removed

- None

### Fixed

- None

## How to test

1. Checkout this branch: `git checkout origin/build/dependencies/10-storybook`
2. Run `yarn` and ensure all dependencies are correctly installed without duplication and that there are no issues related to missing or incompatible dependencies.
3. Run `yarn test` and verify that all tests run and pass.
5. Run `yarn storybook` and verify that the Storybook application runs and behaves as expected.
6. Run `yarn build-storybook` and verify the Storybook build succeeds.
